app/liquid_tags/codepen_tag.rb

```diff
@@ -47,7 +47,7 @@ def parse_link(link)
   def valid_link?(link)
     link_no_space = link.delete(" ")
     (link_no_space =~
-      /^(http|https):\/\/(codepen\.io|codepen\.io\/team)\/[a-zA-Z0-9\-]{1,20}\/pen\/([a-zA-Z]{5,7})\/{0,1}\z/)&.zero?
+      /^(http|https):\/\/(codepen\.io|codepen\.io\/team)\/[a-zA-Z0-9_\-]{1,20}\/pen\/([a-zA-Z]{5,7})\/{0,1}\z/)&.zero?
```

Comment:
this only allows the username to be upto 20 characters long, in my case designpuddle-the-stylish fails the check. Why was this limit set?

Reply:
I believe they have a 20 character limit for usernames now, but my guess is that they added that limit and that there are some accounts that are grandfathered in. Our limit is loosely based on those assumptions, so a PR to adjust it is more than welcome!

Reply:
I created a new account today, and it assigned me a username with 24 characters. I've managed to change the username so I can embed on dev.to but thought I would mention it. I'm happy to create the PR if needed, what would the limit be raised to in that instance?

Reply:
I think 30 characters is a good start. If you could reference this issue that'd be great: https://github.com/thepracticaldev/dev.to/issues/3487